tests/library/inspector/cli-codegen-java.spec.ts

```diff
@@ -83,10 +83,24 @@ test('should print load/save storage_state', async ({ runCLI, browserName }, tes
   await fs.promises.writeFile(loadFileName, JSON.stringify({ cookies: [], origins: [] }), 'utf8');
   const cli = runCLI([`--load-storage=${loadFileName}`, `--save-storage=${saveFileName}`, '--target=java', emptyHTML]);
   const expectedResult1 = `BrowserContext context = browser.newContext(new Browser.NewContextOptions()
-        .setStorageStatePath(Paths.get("${loadFileName.replace(/\\/g, '\\\\')}")));`;
+        .setStorageStatePath(Paths.get(${JSON.stringify(loadFileName)})));`;
   await cli.waitFor(expectedResult1);
 
   const expectedResult2 = `
       context.storageState(new BrowserContext.StorageStateOptions().setPath("${saveFileName.replace(/\\/g, '\\\\')}"))`;
   await cli.waitFor(expectedResult2);
 });
+
+test('should work with --save-har', async ({ runCLI }, testInfo) => {
+  const harFileName = testInfo.outputPath('har.har');
+  const cli = runCLI(['--target=java', `--save-har=${harFileName}`]);
+  const expectedResult = `BrowserContext context = browser.newContext(new Browser.NewContextOptions()
+        .setRecordHarMode(HarMode.MINIMAL)
```

Comment:
I'd prefer it to generate the `routeFromHAR` command for the `--save-har` CLI flag!

Reply:
That would be a separate PR.